### PR TITLE
 [17.0][MIG] stock_quant_package_dimension

### PR DESCRIPTION
**Pull request by ferran-S73**
Standard migration

Depends on:
- https://github.com/OCA/product-attribute/pull/1628
- https://github.com/OCA/stock-logistics-workflow/pull/1618

### CODE REVIEW

File: test-requirements.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+odoo_test_helper
+odoo-addon-stock-quant-package-product-packaging @ git+https://github.com/OCA/stock-logistics-workflow.git@refs/pull/1618/head#subdirectory=stock_quant_package_product_packaging
+odoo-addon-product-packaging-dimension @ git+https://github.com/OCA/product-attribute/pull/1628/head#subdirectory=product_packaging_dimension
```
Comment (rousseldenis) [inline test-requirements.txt]:
```suggestion
odoo-addon-product-packaging-dimension @ git+https://github.com/OCA/product-attribute.git@refs/pull/1628/head#subdirectory=product_packaging_dimension
```

Comment (ferran-S73) [reply test-requirements.txt]: whoops! thanks for your comment, fixed

File: test-requirements.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+odoo_test_helper
```
Comment (traitrantrobz) [inline test-requirements.txt]: Please double-check this file, missing the module [product_logistics_uom](https://github.com/OCA/product-attribute/pull/1625)

File: stock_quant_package_dimension/models/stock_quant_package.py [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+
+    volume_uom_name = fields.Char(
+        # Same as product.packing
+        string="Volume Unit of Measure label",
+        related="volume_uom_id.name",
+        readonly=True,
+    )
+
+    @api.depends("weight_uom_id.name")
+    def _compute_weight_uom_name(self):
```
Comment (traitrantrobz) [inline stock_quant_package_dimension/models/stock_quant_package.py:88]: Please double-check this function. I think you must declare the compute function in the field's definition.

File: stock_quant_package_dimension/readme/DESCRIPTION.md [hunk 4]
```diff
@@ -0,0 +1,2 @@
+This module adds dimension fields on stock packages and an estimated
+weight (in kg).
```
Comment (traitrantrobz) [inline stock_quant_package_dimension/readme/DESCRIPTION.md:2]: I think you should add the file `USAGE.rst` to the description settings and workflow.

**Review by traitrantrobz — CHANGES_REQUESTED** [review]

File: test-requirements.txt (hunk 2 above)
Comment (ferran-S73) [reply test-requirements.txt]: No need to add that module as it is already merged https://github.com/OCA/product-attribute/tree/17.0/product_logistics_uom

I'll check if I have to rebase or smthg

File: stock_quant_package_dimension/models/stock_quant_package.py (hunk 3 above)
Comment (ferran-S73) [reply stock_quant_package_dimension/models/stock_quant_package.py:88]: Not sure why it's not already in the field, I'll fix it, thanks!

**Review by traitrantrobz — APPROVED** [review]
LGTM 👍

**Comment by ferran-S73** [comment]
@OCA/logistics-maintainers

File: stock_quant_package_dimension/models/stock_quant_package.py (hunk 3 above)
Comment (jbaudoux) [reply stock_quant_package_dimension/models/stock_quant_package.py:88]: This fix needs a separate commit as it's not a migration change

Comment (ferran-S73) [reply stock_quant_package_dimension/models/stock_quant_package.py:88]: @jbaudoux done

**Review by jbaudoux — APPROVED** [review]

**Comment by ferran-S73** [comment]
@jbaudoux can you merge please?

**Comment by jbaudoux** [comment]
> @jbaudoux can you merge please?

You need to rewrite commit msg for
`[IMP] stock_quant_package_dimension`
into
`[IMP] stock_quant_package_dimension: pre-commit auto fixes`

**Comment by ferran-S73** [comment]
> > @jbaudoux can you merge please?
> 
> You need to rewrite commit msg for `[IMP] stock_quant_package_dimension` into `[IMP] stock_quant_package_dimension: pre-commit auto fixes`

@jbaudoux done

**Pull request merged**